allure-cucumber4-jvm/src/main/java/io/qameta/allure/cucumber4jvm/AllureCucumber4Jvm.java

```diff
@@ -222,6 +233,14 @@ private void handleTestStepFinished(final TestStepFinished event) {
         }
     }
 
+    private void handleWriteEvent(final WriteEvent event) {
+        lifecycle.addAttachment("Text output", TEXT_PLAIN, TXT_EXTENSION, event.text.getBytes(StandardCharsets.UTF_8));
```

Comment:
can text be null value? I suggest using `Objects.toString(event.text).getBytes(...)` instead

Reply:
Good call. I will change that to `Objects.toString(event.text).getBytes(StandardCharsets.UTF_8)` in the next commit.